python/hsfs/expectation_suite.py

```diff
@@ -389,7 +400,7 @@ def __str__(self) -> str:
         return self.json()
 
     def __repr__(self):
-        return f"ExpectationSuite({self._expectation_suite_name}, {len(self._expectations)} expectations , {self._meta})"
+        return json.dumps(self.to_json_dict())
```

Comment:
This is not really a correct repr function.

In general the repr function should return a string such that you can do the following and get back the identical object (same attributes, data etc. not exactly the same object):
```
obj = ...
new_obj = eval(repr(obj))
```